### ISSUE TITLE
3-dot-menu of file not closable when navigating via keyboard

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/server#10790]
When you Tab through the elements via keyboard and open the 3-dot-menu of a file using Enter, you can not close it again by pressing Enter another time. This is possible using the mouse and should also be possible with the keyboard.

cc @nextcloud/javascript @skjnldsv @juliushaertl what is missing here?

**Comment by MorrisJobke** [comment]
Something for 15 and maybe a backport to stable14 later.

**Comment by skjnldsv** [comment]
Closed in the components.
Will slowly be gone as we move to the component everywhere (already in quite a lot of places :) )

**Comment by jancborchardt** [comment]
@skjnldsv this is specifically about the Files app – which I guess won’t be using components in 17 or for some time?

**Comment by skjnldsv** [comment]
Title was generic, I assumed you meant as a global way.
Well, this interlace with the various vue migration issues we have opened.
I'd say it's not really needed to keep this opened as I'm not sure anyone will work on it until it's easier to know what we're doing in the files code (aka migrating to vue ;) )

But if you still want it opened, you can :)

**Comment by jancborchardt** [comment]
Since it’s an accessbility issue open since 14, I would prefer to keep it open until it is _actually_ fixed in a release. ;) Just so we don’t forget.

**Comment by szaimen** [comment]
Hi, please update to 24.0.8 or better 25.0.2 and report back if it fixes the issue. Thank you!

**Comment by juliushaertl** [comment]
Still valid
